.circleci/config.yml

```diff
@@ -0,0 +1,196 @@
+          command: |
+            eval "$(pyenv init -)"
+            pyenv local $PYTHON
+            python -m pip install virtualenv
+            python -m virtualenv env
+      # Restore cache
+      - restore_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+      - run: *install-dependences-template
+      - save_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - env
+
+      - run: env/bin/coverage run -m unittest discover
+
+
+  test-osx-py37:
+    <<: *osx-tests-template
+    environment:
+      PYTHON: 3.7.0
+      HOMEBREW_NO_AUTO_UPDATE: 1
+
+  test-osx-py36:
+    <<: *osx-tests-template
+    environment:
+      PYTHON: 3.6.5
+      HOMEBREW_NO_AUTO_UPDATE: 1
+
+  #
+  # Windows x64
+  #
+
+  test-win-py38: &win-tests-template
+    executor:
+      name: win/default
+
+    environment:
+      PYTHON: 3.8.0
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Install python
+      - restore_cache:
+          key: nuget-{{ .Environment.CIRCLE_JOB }}
+      - run:
+          name: install python
+          command: nuget install python -Version $env:PYTHON -ExcludeVersion -OutputDirectory .
+      - save_cache:
+          key: nuget-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - .\python
+
+      # Set up virtual environment
+      - run:
+          name: create virtualenv
+          command: |
+            .\python\tools\python.exe --version
+            .\python\tools\python.exe -m pip install virtualenv
+            .\python\tools\python.exe -m virtualenv env
+      # Install requirements
+      - restore_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+      - run: &win-install-dependencies-template
+          name: install dependencies
+          command: |
+            env\Scripts\activate.ps1
+            python --version
+            pip install -r requirements.txt
+            pip install coverage
+      - save_cache:
+          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
+          paths:
+            - env
+
+      # Run tests
+      - run: &win-run-unittests-template
+          name: run unittests
+          command: |
+            env\Scripts\activate.ps1
+            coverage run -m unittest discover
+  test-win-py37:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.7.5
+
+  test-win-py36:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.6.8
+
+  test-win-py35:
+    <<: *win-tests-template
+    environment:
+      PYTHON: 3.5.4
+
+# TODO: add Py3.8 in jobs once ortools (for penaltymodel-mip) supports Py3.8
+workflows:
+  version: 2.1
+  tests:
+    jobs:
+      - test-py38
+      - test-py37
+      - test-py36
+      - test-py35
+      - test-osx-py38
+      - test-osx-py37
+      - test-osx-py36
+      #- test-win-py38
```

Comment:
Shouldn't this run as well?

Reply:
Added to Structural, will add to the others too:
![image](https://user-images.githubusercontent.com/34041130/90254514-b9aced80-ddf7-11ea-8bae-454070476bad.png)